### ISSUE TITLE
New line character problem within {literal} block

### ISSUE THREAD

**Issue by dios-david** [oujesky/closure-templates#1]
When I create a **{literal}** block which has a multiline content, then the generated HTML contains **\n** characters.

**SOY source:**

``` html
{template .test}
    {literal}
        /* some
           multiline
           string */
    {/literal}
{/template}
```

**Generated PHP source:**

``` php
  public static function test($opt_data = null, $opt_ijData = null) {
    $output = '\n        /* some\n           multiline\n           string */\n    ';
    return new \Goog\Soy\SanitizedHtml($output);
  }
```

**Rendered HTML source:**

``` html
\n        /* some\n           multiline\n           string */\n    
```

So the problem is when setting the value of **$output**, the value is surrounded by _single quotes_, so the _new line character_ will be treated as text instead of as a special character. Changing _single quotes_ to _double quotes_ or replacing **\n** with **PHP_EOL** can fix this issue.

**Comment by oujesky** [comment]
Solved by da917757763628238577ad669cea19a90cbb4d01

Please try downloading new build at https://github.com/oujesky/closure-templates/releases/tag/latest

**Comment by dios-david** [comment]
Thank you Miroslav, it seems to be working well :)
